src/EditorFeatures/Core/InheritanceMargin/InheritanceMarginServiceHelpers.cs

```diff
@@ -18,6 +18,24 @@ namespace Microsoft.CodeAnalysis.InheritanceMargin
 {
     internal static class InheritanceMarginServiceHelper
     {
+        private static readonly SymbolDisplayFormat s_displayFormat = new(
+                globalNamespaceStyle: SymbolDisplayGlobalNamespaceStyle.OmittedAsContaining,
+                typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameAndContainingTypes,
+                genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters,
+                memberOptions:
+                    SymbolDisplayMemberOptions.IncludeParameters |
+                    SymbolDisplayMemberOptions.IncludeContainingType |
+                    SymbolDisplayMemberOptions.IncludeExplicitInterface,
```

Comment:
we might also want the return type.  or did you think that was too much?

Reply:
I feel when I click a method, then I can see its return type directly in the editor, and since all its parent and derived methods should have the same return type, it feels like unnecessary.

Reply:
i can see that perspective, and i don't disagree with it.  For my benefit though, could you try this and supply a screenshot?  If it does feel very redundant, i'm fine to keep leaving it out.  I'd just like to take a look thouhg.

Note: teh same holds true for things like parameter types, but it looks like we're leaving that in.  I think it's the inconsistency that makes it feel weird to me.

Reply:
Sure, let me find an example, actually I now feel the parameters names are not useful,
For example:
Now:
![image](https://user-images.githubusercontent.com/24360909/115619682-0d39da00-a2a9-11eb-81fe-9eddcbb978b3.png)

If I remove the parameters:
![image](https://user-images.githubusercontent.com/24360909/115620557-268f5600-a2aa-11eb-9ff0-c942ccb17a22.png)

If I include the parameters and return value:
![image](https://user-images.githubusercontent.com/24360909/115621057-db297780-a2aa-11eb-9b68-e45c10bcb2b6.png)

Reply:
I think removing the parameters and the return type looks the best overall.

Reply:
Agree, it's the "quickest" overview for what you're navigating too

Reply:
Done. parameter is reomved.